src/javascript/common/header.js

```diff
@@ -13,9 +13,11 @@ const Header = (function () {
     const userMenu = function() {
         if (!Client.is_logged_in()) {
             $('#main-login').removeClass('hidden');
+            $('#main-signup').removeClass('hidden');
```

Comment:
What do you think about `$('#main-login, #main-signup').removeClass('hidden');`?